databricks/koalas/plot.py

```diff
@@ -1243,6 +1315,38 @@ def line(self, x=None, y=None, **kwargs):
         See Also
         --------
         matplotlib.pyplot.plot : Plot y versus x as lines and/or markers.
+
+        Examples
+        --------
+
+        .. plot::
+            :context: close-figs
+
+            The following example shows the populations for some animals
+            over the years.
+
+            >>> df = ks.DataFrame({
+            ...    'pig': [20, 18, 489, 675, 1776],
+            ...    'horse': [4, 25, 281, 600, 1900]
+            ...    }, index=[1990, 1997, 2003, 2009, 2014])
+            >>> lines = df.plot.line()
+
+        .. plot::
+           :context: close-figs
+
+           An example with subplots, so an array of axes is returned.
```

Comment:
different indentation from others

Reply:
@harupy Eagle eye! thanks for find my mistake out. i fixed it. Thanks 😃